.github/workflows/docs-build.yml

```diff
@@ -36,3 +36,51 @@ jobs:
           git add -f docs static/statuses.json static/arguments.yml static/classes.yml
           git commit -m "docs: build docs" || true
           git push
+      - name: Set up node js
+        uses: actions/setup-node@v2
+        with:
+          node-version: 15
+      - name: Run docs
+        run: |
+          cd docs
+          yarn
+          yarn develop & npx wait-on http://localhost:8000
+      - name: Run axe checks on examples of component docs
+        run: |
+          npm install -g @axe-core/cli
+          files=($(grep -r -l "Examples" docs/content/components))
+          declare -a needs_to_be_fixed=(
+            autocomplete 
+            autocompleteitem
+            dropdown
+            dropdownmenu
+            markdown
+            navigationtab
```

Comment:
Excluding the check for component pages that are failing violations at this starting point. This will fail build for all new new components with bad examples.

Reply:
Instead of building in escape hatches for some components, can we fix those issues in separate PRs and then return to this PR and merge it?

Reply:
I agree 100% that we should make sure to address these. My concern is that it may take some time to look into each of these and meanwhile more components may get merged in. I was thinking we should get this action out the door, and take time to look into each of these carefully